crates/codegen/parser/runtime/src/kinds.rs

```diff
@@ -76,14 +73,10 @@ pub enum NodeLabel {
     Operand,
     LeftOperand,
     RightOperand,
-    LeadingTrivia,
-    TrailingTrivia,
-    // Used for testing this crate, this is generated in the client code
-    Label1,
-    Label2,
-    Label3,
 }
```

Comment:
Not sure I understand why are we removing `LeadingTrivia` and `TrailingTrivia` only from here?